src/modules/UI/scenes/CryptoExchange/CryptoExchangeQuoteProcessingScreenComponent.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import React, { Component } from 'react'
+import { ActivityIndicator, View } from 'react-native'
+
+import s from '../../../../locales/strings.js'
+import { CryptoExchangeQuoteProecessingSceneStyles as styles } from '../../../../styles/indexStyles'
+import Gradient from '../../../UI/components/Gradient/Gradient.ui'
+import FormattedText from '../../components/FormattedText'
+import SafeAreaView from '../../components/SafeAreaView'
+
+type Props = {
+  shiftPendingTransaction: boolean,
```

Comment:
These props are not used anywhere in the code?!

Reply:
removed and fixed